agreement_sale/views/agreement.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+
+<odoo>
+
+    <menuitem id="agreement.agreement_menu" action="agreement.agreement_action" 
```

Comment:
@max3903 Is it fine under the sale order configuration?

Reply:
Yes